### PR TITLE
Add JSON for Zenodo

### PR DESCRIPTION
**Pull request by palday**

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+  {
+    "license": "other-open",
```
Comment (ararslan) [inline .zenodo.json:2]: Shouldn't this be MIT?

Comment (palday) [reply .zenodo.json:2]: still in draft because I've been trying to find their list of possible enum values for this field 😄

Comment (palday) [reply .zenodo.json:2]:
```suggestion
    "license": "mit-license", 
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "upload_type": "software", 
+    "creators": [
+        {
+            "affiliation": "Beacon Biosignals", 
+            "name": "Alday, Phillip M."
+            "orcid": "0000-0002-9984-5745"
+        }, 
+        {
+            "affiliation": "Beacon Biosignals", 
+            "name": "Kleinschmidt, Dave K."
```
Comment (kleinschmidt) [inline .zenodo.json:13]:
```suggestion
            "name": "Kleinschmidt, Dave F."
```

**Review by kleinschmidt — APPROVED** [review]

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            "orcid": "0000-0002-7442-2762"          
+        }, 
+        {
+            "affiliation": "Beacon Biosignals", 
+            "name": "Arslan, Alexander"
```
Comment (ararslan) [inline .zenodo.json:18]:
```suggestion
            "name": "Arslan, Alexander R."
```
Everybody else has middle initials, feeling left out

**Review by ararslan — APPROVED** [review]

**Pull request merged**